src/main/java/edu/cornell/kfs/fp/document/validation/impl/CuDisbursementVoucherDocumentPreRules.java

```diff
@@ -1,67 +1,242 @@
 import org.kuali.kfs.fp.document.validation.impl.DisbursementVoucherDocumentPreRules;
+import org.kuali.kfs.pdp.PdpParameterConstants;
+import org.kuali.kfs.pdp.businessobject.PaymentNoteText;
 import org.kuali.kfs.sys.KFSConstants;
 import org.kuali.kfs.sys.businessobject.PaymentSourceWireTransfer;
 import org.kuali.kfs.sys.context.SpringContext;
+import org.kuali.kfs.sys.service.impl.KfsParameterConstants;
 import org.kuali.rice.core.api.config.property.ConfigurationService;
+import org.kuali.rice.core.api.util.type.KualiInteger;
 
+import edu.cornell.kfs.fp.CuFPConstants;
+import edu.cornell.kfs.fp.document.CuDisbursementVoucherConstants;
 import edu.cornell.kfs.sys.CUKFSKeyConstants;
 
+import org.kuali.rice.coreservice.framework.parameter.ParameterService;
+import org.kuali.rice.krad.document.Document;
+
 /**
  * Checks warnings and prompt conditions for dv document.
  */
 public class CuDisbursementVoucherDocumentPreRules extends DisbursementVoucherDocumentPreRules {
 
-    /**
-     * This method returns true if the state of all the tabs is valid, false otherwise.
-     *
-     * @param dvDocument submitted disbursement voucher document
-     * @return Returns true if the state of all the tabs is valid, false otherwise.
-     */
-    @SuppressWarnings("deprecation")
-    protected boolean checkWireTransferTabState(DisbursementVoucherDocument dvDocument) {
-        boolean tabStatesOK = true;
-
-        PaymentSourceWireTransfer dvWireTransfer = dvDocument.getWireTransfer();
-
-        // if payment method is CHECK and wire tab contains data, ask user to clear tab
-        if ((StringUtils.equals(KFSConstants.PaymentSourceConstants.PAYMENT_METHOD_CHECK, dvDocument.getDisbVchrPaymentMethodCode()) || StringUtils.equals(KFSConstants.PaymentSourceConstants.PAYMENT_METHOD_DRAFT, dvDocument.getDisbVchrPaymentMethodCode())) && hasWireTransferValues(dvWireTransfer)) {
-            String questionText = SpringContext.getBean(ConfigurationService.class).getPropertyValueAsString(CUKFSKeyConstants.QUESTION_CLEAR_UNNEEDED_WIRE_TAB);
-
-            boolean clearTab = super.askOrAnalyzeYesNoQuestion(KFSConstants.DisbursementVoucherDocumentConstants.CLEAR_WIRE_TRANSFER_TAB_QUESTION_ID, questionText);
-            if (clearTab) {
-                // NOTE: Can't replace with new instance because Foreign Draft uses same object
-                clearWireTransferValues(dvWireTransfer);
+  /**
+   * Executes pre-rules for Disbursement Voucher Document
+   *
+   * @param document submitted document
+   * @return true if pre-rules execute successfully
+   * @see org.kuali.rice.kns.rules.PromptBeforeValidationBase#doRules(org.kuali.rice.kns.document.MaintenanceDocument)
+   */
+  @Override
+  public boolean doPrompts(Document document) {
+    boolean preRulesOK = super.doPrompts(document);
+
+    DisbursementVoucherDocument dvDocument = (DisbursementVoucherDocument) document;
+    checkSpecialHandlingIndicator(dvDocument);
```

Comment:
It does seem to call it. Perhaps there used to be a custom one that has been removed since the last time I worked on this? Regardless, I'll change this then.